### PR TITLE
Typings

### PR DESCRIPTION
**Pull request by blvdmitry**
- Updated typings
- Specified type entry point and published files in package.json

### CODE REVIEW

File: localstory.d.ts [hunk 1]
```diff
@@ -4,20 +4,21 @@ declare module "localstory" {
...
+    values: () => (T[keyof T])[];
   };
 
-  export default function(
-    store: any,
-    ns?: string,
-    options?: LSOptions
-  ): LSStore;
+  type LSCreateStore<T extends object = { [key: string]: unknown }> =
+    (store: T, ns?: string, options?: LSOptions) => LSStore<T>;
```
Comment (ricardobeat) [inline localstory.d.ts]: Usually you'll pass `window.localStorage` or `sessionStorage` as the store. How are you using this to set the type - shouldn't it take a generic so you can do:

```js
localstory<MyStoreShape>(window.localStorage, 'myNamespace')
```

Comment (blvdmitry) [reply localstory.d.ts]: Yep, exactly this. `createStore: LSCreateStore` is the `localstory` function in your code snippet. It takes the store shape as a generic which should extend an object. 

Also after looking at this once again - probably it should be also defaulting to this kind object with unknown values when used without passing the generic type manually.

Comment (blvdmitry) [reply localstory.d.ts]: @ricardobeat Updated it to expose a generic type to the users, missed it before as you've mentioned + improved method typings:

image

**Comment by ricardobeat** [comment]
@blvdmitry published as 0.9.1. Thanks!

**Pull request merged**